.prout.json

```diff
@@ -1,7 +1,14 @@
 {
     "checkpoints": {
-        "PROD": {
-            "url": "https://frontend.gutools.co.uk/deploys?stage=PROD&pageSize=50",
+        "FRONTS-PROD": {
+            "url": "https://www.theguardian.com/uk",
+            "overdue": "30M",
+            "messages": {
+                "seen": "prout/seen.md"
+            }
+        },
+        "ADMIN-PROD": {
+            "url": "https://frontend.gutools.co.uk/login",
             "overdue": "30M",
             "messages": {
                 "seen": "prout/seen.md"
```

Comment:
Don't need to fix it with this PR, but later we may want to remove this 2nd `messages` conf to avoid getting duplicate messages (ie two copies of https://github.com/guardian/frontend/blob/main/prout/seen.md) - I'm not sure if Prout will de-duplicate the messages.